Source/UI/Buttons/RelationshipButton.swift

```diff
@@ -211,27 +205,29 @@ class RelationshipButton: IconButton {
     }
 
     required init?(coder aDecoder: NSCoder) {
```

Comment:
I have been marking these unavailable, like this:
```suggestion
    @available(*, unavailable)
    required init?(coder aDecoder: NSCoder) {
```

I think that will make a compile time error if anyone ever tries to call this function.

Reply:
Done